android/app/src/main/java/com/dismoi/scout/accessibility/BackgroundService.kt

```diff
@@ -112,16 +113,8 @@ class BackgroundService : AccessibilityService() {
       info.getChild(0).className.toString() == "android.widget.EditText"
   }
 
-  private fun overlayIsActivated(applicationContext: Context): Boolean {
-    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-      canDrawOverlays(applicationContext)
-    } else {
-      false
-    }
-  }
-
-  private fun isWindowChangeEvent(event: AccessibilityEvent): Boolean {
-    return AccessibilityEvent.eventTypeToString(event.eventType).contains("WINDOW")
+  fun isLauncherActivated(packageName: String): Boolean {
+    return "com.android.launcher3" == packageName
```

Comment:
This will work only with stock Android launcher I guess ?

Reply:
Yes we need to check if the app for the launcher differs based on different android versions or brand. Opening a PR for this. Might put this in a list or array